newlib/libc/machine/riscv/memcpy-asm.S

```diff
@@ -9,7 +9,8 @@
    http://www.opensource.org/licenses.
 */
 
-#include "./rv_memcpy.h"
+#define _RV_MEMCPY_
```

Comment:
I'm not sure these defines are required -- can't rv_string.h simply always define the C/ASM selectors for all functions?